### DISCUSSION TITLE
[QoL] ZG additional Idol drop from Hakkar according to SoM Classic

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#4171] [Suggestions]
Season of Mastery Classic changed some drop / loot mechanics for ZG.
One change includes the addition of an 100% Idol drop to Hakkars loot table, in addition to the already dropping Idols from Jindo and Mandokir.

this would be a nice QoL addition to CC.

Source: https://classic.wowhead.com/news/classic-season-of-mastery-hakkar-bonus-loot-326171

**Comment by Si1ker** [comment]
![image](https://user-images.githubusercontent.com/61223313/195344105-26116793-9b13-4fb2-aec4-b6114e9b5d66.png)
Source: https://us.forums.blizzard.com/en/wow/t/season-of-mastery-hakkar-bonus-loot/1194036

**Comment by Cooties-cc** [comment]
This is pretty much a custom change to how the raid works, but I understand wanting more idols to be obtained. Since until 70, all [most] classes are going to have to use ZG enchants on head slot, which is 2 chromie patch cycles during TBC. I would like it better if they were to do a temporary vendor/NPC situation where we could trade/turnin something for a ZG idol. Of course it would be no means cheap, and nothing that wouldn't come from the raid itself, as gaining a raid drop outside a raid is sorta a no-no. Perhaps they could have it so the Heart of Hakkar could be traded or turned in to an NPC for an idol. The rewards from the heart kinda suck already compared to other items, and it comes directly from a Hakkar kill. 2 birds 1 stone, no permanent changes.

**Reply by balleny** [reply]
we arent discussing the hardmode additional loot from the skipped priestess.
Its just about adding the Idol Hakkar drops now ALWAYS, also to CC.

exchange e.g. for the heart could be difficult technically: I am not sure if you can receive the heart again into your inventory, if you done the quest already.
Some other nice alternative I could imagine (but also custom) would be the option to exchange 100 of the [Zandalar Honor Tokens](https://db.rising-gods.de/?item=19858) for an Idol.

**Reply by Cooties-cc** [reply]
Yes I know it was only about hakkar getting an idol, but it's still custom to add an idol to hakkar's loot table. I sorta forgot heart is unique 1 time item, and was only thinking about something that _always_ dropped from him people could exchange. With that in mind, yeah honor tokens seem to be the next best thing. However 100 seems a bit too steep. The shoulder enchant only costs 30, and is 1 slot. The idol covers 2 slots, so you'd need twice that amount (per person). There's also the problem that honor tokens can be obtained without ever killing a boss, which negates bringing hakkar into the mix, as the main idea was to get +1 idol from a boss kill in ZG. 

What if this could be linked to killing hakkar hardmode? Say you kill hakkar with all bosses alive still, you got +1 idol? Or if only the idol bosses were left alive when you killed hakkar, hakkar dropped +1 idol. They could add onto the bonus loot tables which could be removed later when idol enchants become useless at 70. This means too if players wanted extra idols, they'd have to work extra hard to do a hardmode for +3, else just the normal +2 runs.

**Reply by balleny** [reply]
we usually do hakkar in a 1 or 2 priestess alive hardmode.
If 3-4 priestess alive hardmode would offer 3-4 additional idols it would be a great incentive - but also more effort to script regarding the loot table.
